evalml/tests/data_checks_tests/test_target_leakage_data_check.py

```diff
@@ -96,15 +100,14 @@ def test_target_leakage_data_check_input_formats():
                                       details={"column": "d"}).to_dict()],
         "errors": []
     }
-
     # test X as ww.DataTable, y as ww.DataColumn
     assert leakage_check.validate(ww.DataTable(X), ww.DataColumn(y)) == expected_messages
 
-    #  test y as list
+    # test y as list
     assert leakage_check.validate(X, y.values) == expected_messages
 
     # test X as np.array
-    assert leakage_check.validate(X.to_numpy(), y) == {
+    assert leakage_check.validate(X.to_numpy().astype(float), y) == {
```

Comment:
Type info is lost here, so need to manually convert to float: https://github.com/alteryx/woodwork/issues/456